Polygon_mesh_processing/examples/Polygon_mesh_processing/hausdorff_bounded_error_distance_example.cpp

```diff
@@ -0,0 +1,225 @@
+#include <CGAL/Bbox_3.h>
```

Comment:
This example is too complicated and should be moved to test. A simpler example should be added then.

Reply:
Done.